docs/cost/ids/core.md

```diff
@@ -0,0 +1,271 @@
+# Core standard
+
+<style>
+.wy-nav-content {
+  max-width: 1200px;
+}
+</style>
+
+This page documents a mapping from the 'core' CoST IDS elements to OC4IDS fields and codes. The organization of the mapping tables reflects the structure of the CoST IDS.
```

Comment:
In case I close https://github.com/open-contracting/standard-maintenance-scripts/issues/10 (done for extensions, but not for standards or profiles), let's use double quotes for colloquial quotes.

```suggestion
This page documents a mapping from the "core" CoST IDS elements to OC4IDS fields and codes. The organization of the mapping tables reflects the structure of the CoST IDS.
```